WalletConnectSwift.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |spec|
   spec.license      = { :type => "MIT", :file => "LICENSE" }
   spec.author             = { "Andrey Scherbovich" => "andrey@gnosis.io", "Dmitry Bespalov" => "dmitry.bespalov@gnosis.io" }
   spec.cocoapods_version = '>= 1.4.0'
-  spec.platform     = :ios, "12.0"
+  spec.platform     = :ios, "11.0"
```

Comment:
please also change the `spec.version` to `1.0.1` on line 2

Reply:
Great, changed (force-push)